lib/builder/builder.go

```diff
@@ -259,19 +256,10 @@ func (b *Builder) Vendor(ctx context.Context, dir string) (io.ReadCloser, error)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	manifestPath := filepath.Join(dir, defaults.ResourcesDir, "app.yaml")
-	// If manifest filename is empty, it means it was auto-generated
-	// out of a Helm chart so write the generated manifest to the
-	// vendor directory as well.
-	if b.manifestFilename == "" {
-		data, err := yaml.Marshal(b.Manifest)
-		if err != nil {
-			return nil, trace.Wrap(err)
-		}
-		err = ioutil.WriteFile(manifestPath, data, defaults.SharedReadMask)
-		if err != nil {
-			return nil, trace.Wrap(err)
-		}
+	manifestPath := filepath.Join(dir, defaults.ResourcesDir, defaults.ManifestFileName)
+	err = writeManifest(manifestPath, b.Manifest)
```

Comment:
Do we actually update base version to the selected runtime version in the manifest anywhere? I can't see it, so I'm not sure just copying the manifest does anything?